### PR TITLE
Changes for converting Slack team to workspace

### PR DESCRIPTION
**Pull request by vasujain**
screen shot 2018-06-19 at 8 14 00 pm

### CODE REVIEW

**Comment by vasujain** [comment]
I signed it

File: tutorials/slack-slash-command-app-engine-java.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ page](https://api.slack.com/apps) and click **Create new app**.
 ### Install the command to your team
```
Comment (JayLoomis) [inline tutorials/slack-slash-command-app-engine-java.md]: Should the heading also be changed to "workspace"?

Comment (vasujain) [reply tutorials/slack-slash-command-app-engine-java.md]: Sure, I guess that makes sense. Will update the PR.

**Comment by vasujain** [comment]
I guess you have to merge it @JayLoomis .  I don't have `write access`.

**Pull request merged**